go/teams/teams.go

```diff
@@ -579,13 +614,16 @@ func (t *Team) rotateBoxes(ctx context.Context, memSet *memberSet) (*PerTeamShar
 	return t.keyManager.RotateSharedSecretBoxes(deviceEncryptionKey, memSet.recipients)
 }
 
-func (t *Team) sigPayload(sigMultiItem libkb.SigMultiItem, secretBoxes *PerTeamSharedSecretBoxes, lease *libkb.Lease) libkb.JSONPayload {
+func (t *Team) sigPayload(sigMultiItem libkb.SigMultiItem, secretBoxes *PerTeamSharedSecretBoxes, lease *libkb.Lease, aux libkb.JSONPayload) libkb.JSONPayload {
 	payload := make(libkb.JSONPayload)
 	payload["sigs"] = []interface{}{sigMultiItem}
-	payload["per_team_key"] = secretBoxes
+	if secretBoxes != nil {
+		payload["per_team_key"] = secretBoxes
+	}
 	if lease != nil {
 		payload["downgrade_lease_id"] = lease.LeaseID
 	}
+	payload["aux"] = aux
```

Comment:
oh, this isn't what i expected.  can you rename the `aux` parameter `payload`, and just have the caller pass in the preloaded `payload`, shouldn't that work?